File: Python/2566.py
## 최댓값
"""import sys

arr = [list(map(int, sys.stdin.readline().split())) for _ in range(9)]

ans = -1
x, y = 0, 0
for i in range(9):
    for j in range(9):
        if arr[i][j] > ans:
            ans = arr[i][j]
            x, y = i + 1, j + 1
print(ans)
print(x, y)"""

import sys

input = sys.stdin.readline

arr = []
for _ in range(9):
    arr += list(map(int, input().split()))

idx = arr.index(max(arr))
print(max(arr))
print(idx // 9 + 1, idx % 9 + 1)

File: JAVA/src/_2566.java
import java.util.Scanner;

public class _2566 {
    public static void main(String[] args) {
        int arr[][] = new int[9][9];
        Scanner sc = new Scanner(System.in);

        int max = -1, x = 0, y = 0;
        for (int i = 0; i < 9; i++) {
            for (int j = 0; j < 9; j++) {
                arr[i][j] = sc.nextInt();
                if (arr[i][j] > max)
                {
                    max = arr[i][j];
                    x = i;
                    y = j;
                }
            }
        }
        
        System.out.println(max);
        System.out.printf("%d %d", x + 1, y + 1);
    }
    
}